x/delegation/types/genesis.go

```diff
@@ -197,7 +197,7 @@ func (gs GenesisState) ValidateStakerList() error {
 
 func (gs GenesisState) ValidateUndelegations() error {
 	validationFunc := func(_ int, undelegation UndelegationRecord) error {
-		err := ValidateIDAndOperator(undelegation.StakerID, undelegation.AssetID, undelegation.OperatorAddr)
+		err := ValidateIDAndOperator(undelegation.StakerId, undelegation.AssetId, undelegation.OperatorAddr)
```

Comment:
We only use `AssetId` in the code for proto-generation to maintain compatibility with GRPC. I don't think it is necessary to change all `AssetID` to `AssetId`, because it will introduce too many changes to the code.